docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,139 @@
+# Running the Airy Core Plaform in production
+
+
+The cloud providers part can be left as TODO in the first pr. It's enough to cover all the requirements (size of a kubernetes cluster, how to get the code up and running in the cluster, how to run the services (redis, postgres, kafka, zookeeper) both in the same kubernets cluster or how to configure the apps so that the system works)
+
+## Architecture
+
+The Airy Core Platform is messaging platform, based on microservices and comprised of a backend and frontend system. 
```

Comment:
```suggestion
The Airy Core Platform is a microservice based messaging platform comprised of a backend and frontend system. 
```

Reply:
I wouldn't use the work microservice as we don't really buy on the philosophy much (take api-communication as an example)

Reply:
ok, will remove microservices